XIVComboVX/Config/PluginConfiguration.cs

```diff
@@ -98,7 +98,7 @@ public class PluginConfiguration: IPluginConfiguration {
 		0,
 		30
 	)]
-	public float GunbreakerGnashingStrikeCooldownGnashingFang { get; set; } = 20;
+	public float GunbreakerGnashingStrikeCooldownGnashingFang { get; set; } = 17;
```

Comment:
Just a heads up, this will not affect anyone who's already used a version with this setting, whether or not they've adjusted it from the (old) default value. The setting and its value are already baked into the saved config file.